src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -844,5 +844,50 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
             }
         }
         #endregion
+
+        #region Proxy tests
```

Comment:
FYI This source file is getting really big now as we continue to add regions.  I plan to break apart this file soon.